pkg/distro/rhel8/ami.go

```diff
@@ -24,18 +24,14 @@ func amiImgTypeX86_64(rd distribution) imageType {
 		buildPipelines:      []string{"build"},
 		payloadPipelines:    []string{"os", "image"},
 		exports:             []string{"image"},
-		basePartitionTables: ec2BasePartitionTables,
+		basePartitionTables: getEc2PartitionTables(rd.osVersion, rd.isRHEL()),
```

Comment:
So previously, the `ami` would always use the latest partitioning scheme, regardless of the release version. This won't be true any more.

Reply:
intentional change, resolving...